### PR TITLE
errors: More informative error message if wrong table is specified

### PR DESCRIPTION
**Pull request by igor-savin-ht**
for relation.

### CODE REVIEW

File: lib/relations/RelationProperty.js [hunk 1]
```diff
@@ -182,7 +182,9 @@ function createPaths(refs, modelClassResolver) {
     const modelClass = modelClassResolver(ref.tableName);
 
     if (!modelClass) {
-      throw new ModelNotFoundError();
+      const error = new ModelNotFoundError();
+      error.tableName = ref.tableName;
```
Comment (koskimas) [inline lib/relations/RelationProperty.js]: This property should be added to the class and passed to the constructor:

```js
class ModelNotFoundError extends Error {
  constructor(tableName) {
    super();
    this.tableName = tableName;
  }
}
```

**Comment by koskimas** [comment]
Thanks for the fix! I had one small suggestion. If you fix that, I'll merge this.

File: lib/relations/RelationProperty.js (hunk 1 above)
Comment (fl0w) [reply lib/relations/RelationProperty.js]: May I quickly ask why/the reason for this?

**Comment by igor-savin-ht** [comment]
@koskimas Done, thank you!

**Comment by koskimas** [comment]
Awesome! Thank you!

**Pull request merged**